include/irr/asset/IRenderpassIndependentPipeline.h

```diff
@@ -28,32 +28,37 @@ enum E_PRIMITIVE_TOPOLOGY
     EPT_PATCH_LIST = 10
 };
 
-enum E_VERTEX_INPUT_RATE
+enum E_VERTEX_INPUT_RATE : uint32_t
 {
     EVIR_PER_VERTEX = 0,
     EVIR_PER_INSTANCE = 1
 };
 
 struct SVertexInputAttribParams
 {
+    uint32_t binding;
     E_FORMAT format;
     uint32_t offset;
```

Comment:
call this field `relativeoffset` also could fit this and `binding` in a `uint16_t` because there are limits for each value much lower than 64k

Reply:
>also could fit this and binding in a uint16_t because there are limits for each value much lower than 64k

most of these are 32bits because i'm trying to make them look as close as possible to Vulkan's structs

Reply:
we won't be able to `memcpy` this into a vulkan struct anyway cause `E_FORMAT` doesn't match `VkFormat`

Reply:
@Crisspl ?

Reply:
renamed long ago